src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -130,33 +134,52 @@ public ModuleLoader(IDiscordClient client) {
 	public static List<Class<? extends IModule>> getModules() {
 		return modules;
 	}
-
+	
 	/**
 	 * Manually loads a module.
 	 *
 	 * @param module The module to load.
 	 * @return true if the module was successfully loaded, false if otherwise. Note: successful load != successfully enabled
 	 */
 	public boolean loadModule(IModule module) {
-                if (!loadedModules.contains(module) && !canModuleLoad(module)) {
+		return loadModule(module, false, true);
+	}
+	
+	private boolean loadModule(IModule module, boolean dryRun, boolean registerComponents) {
```

Comment:
While that is the case, we do not do this anywhere else so it would be inconsistent